src/contrib/mlir/core/pass/ng_dialect_builder.cpp

```diff
@@ -117,8 +96,9 @@ namespace
 
         // Generic op lowerer to ng dialect.
         // Simply maps ngraph tensors to values and generate an OP. No op-specific logic.
+        // Use inNum when mlir OP needs less input than its corresponding ngraph OP.
         template <typename Op>
-        mlir::Operation* createGenericOp(const ngraph::Node* ngNode);
+        mlir::Operation* createGenericOp(const ngraph::Node* ngNode, int inNum = 0);
```

Comment:
use -1 as a default value instead. I could want to create a generic op with zero operands.